### PR TITLE
AO3-6667 Languages page performance improvements

### PR DESCRIPTION
**Pull request by brianjaustin**
# Pull Request Checklist

* [X] Have you read ["How to write the perfect pull request"](https://github.blog/2015-01-21-how-to-write-the-perfect-pull-request/)?
* [X] Have you read the [contributing guidelines](https://github.com/otwcode/otwarchive/blob/master/CONTRIBUTING.md)?
* [X] Have you added [tests for any changed functionality](https://github.com/otwcode/otwarchive/wiki/Automated-Testing)?
* [X] Have you added the [Jira](https://otwarchive.atlassian.net) issue number
  as the *first* thing in your pull request title (e.g. `AO3-1234 Fix thing`)
* [X] Do you have fewer than 5 pull requests already open? If not, please wait
  until they are reviewed and merged before creating new pull requests.

## Issue

https://otwarchive.atlassian.net/browse/AO3-6667

## Purpose

* Fetch language counts from Elasticsearch instead of the primary DB
* Round counts a bit to make temporary discrepancies (in the ones or tens place) less impactful. The counts don't need to be exact anyway
* Cache counts for 1 day to further reduce load

### CODE REVIEW

File: config/locales/views/en.yml [hunk 1]
```diff
@@ -528,6 +528,15 @@ en:
       more_link:
         one: "%{count} more user"
         other: "%{count} more users"
+  languages:
+    index:
+      add: Add a language
+      description: Work Languages
+      edit: Edit
+      suggest: Suggest a language
```
Comment (sarken) [inline config/locales/views/en.yml:536]: Could you keep the original capitalization from the navigation actions? Our navigation should always be title case. (You can, however, drop the "a" if you want.) It also wouldn't hurt to group them under `navigation` like we do elsewhere to give Translation a bit more context, e.g. https://github.com/otwcode/otwarchive/blob/653c19c250d9e35b2ea33e86a51b7b28c1e4c975/config/locales/views/en.yml#L356-L361

File: app/views/languages/index.html.erb [hunk 2]
```diff
@@ -1,38 +1,37 @@
 <!--Descriptive page name, messages and instructions-->
-<h2 class="heading"><%= ts('The Archive supports these languages') %></h2>
+<h2 class="heading"><%= t(".description") %></h2>
```
Comment (sarken) [inline app/views/languages/index.html.erb:2]: Can we call this `page_heading`? That way if we ever have to add a description to the page (i.e., a paragraph under the heading), we don't need to rename. It's also not a bad idea to have a consistent naming scheme for bits that appear on all pages, and it looks like that one is used pretty frequently.

**Review by sarken — COMMENTED** [review]
Just took a quick look and it's possible the failing tests need `work_search: true` to set up the indexes, like https://github.com/otwcode/otwarchive/blob/653c19c250d9e35b2ea33e86a51b7b28c1e4c975/spec/controllers/collection_participants_controller_spec.rb#L113

File: config/locales/views/en.yml (hunk 1 above)
Comment (brianjaustin) [reply config/locales/views/en.yml]: Sounds good! I think keeping the `a` sounds better but I can remove it if that's the standard

**Review by sarken — APPROVED** [review]

**Pull request merged**